pyproject.toml

```diff
@@ -167,6 +167,11 @@ invoke = "~1.7.1"
 mkdocs-include-markdown-plugin = "~3.6.0"
 # Rendering docs to HTML
 mkdocs = "~1.3.0"
+# Material for mkdocs theme
+mkdocs-material = "~8.4.2"
```

Comment:
Should this be ==8.4.2 instead of ~8.4.2? Since the docs/requirements.txt is expecting an exact version.